### ISSUE TITLE
Future consideration

### ISSUE THREAD

**Issue by ffes** [hansworst24/Ticz#28]
In the README you write:

>  so it might not be impossible that Ticz returns Android-style :)

Don't know if you still consider the possibility to make Ticz for Android. If so, please take a look at [Xamarin](https://www.xamarin.com/). This is a multi platform framework to create Android, UWP and iOS apps from the same C# code base. It is available for free in Visual Studio Community Edition.

The advantage is that you already have a lot of .NET code to get things done and it allows you to create both the Android app as well as a UWP (great for the Windows 10 desktop, live tiles, etc).

At the moment Ticz seems the only domoticz app for Windows 10, so I hope someone will keep it alive!

Just a thought ;-)

**Comment by hansworst24** [comment]
@ffes : You are right. I've looked briefly at Xamarin. Although I understand the concept and quickly tried some things, I think it will take me a huge amount of time to re-develop Ticz with Xamarin stuff. First, I need to learn C# 😄 , second I need to replace every control with a Xamarin control. I feel that with Xamarin, although you can create a working cross-platform app, it will never be as good-looking as Ticz currently is 😃 Therefore I prefer to keep Ticz as it is, and hopefully someone else will feel the need to keep it updated !
